irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -29,7 +29,8 @@ namespace iroha {
     bool PostgresWsvCommand::insertRole(const std::string &role_name) {
       try {
         transaction_.exec("INSERT INTO role(role_id) VALUES ("
-                          + transaction_.quote(role_name) + ");");
+                          + transaction_.quote(role_name)
+                          + ");");
       } catch (const std::exception &e) {
```

Comment:
In PostgreSQL there is a thing named [*prepared statements*](https://stackoverflow.com/questions/31833322/how-to-prepare-statements-and-bind-parameters-in-postgresql-for-c#31884939). It would suggest to use them, because of:
1. templating - no need to write the same query twice or more 
2. safety - no need in quoting all parameters (thus making the code cleaner) to prevent SQLinj.
    > Many types of SQL injection attacks can be eliminated by disabling literals, effectively requiring the use of prepared statements -- Wikipedia

Reply:
Yeah, we know. Probably in other pr

Reply:
Good point, I suggest adding TODO about it in code.